README.md

```diff
@@ -133,10 +134,39 @@ CycleTimeMs - Cycle time of the application.  All tags will be posted at this sp
 
 PostDuplicateTagValues - Controls when datapoints are logged.  If set to true, datapoints will always be cyclicly logged. If set to false, datapoints will only be logged on change of value.
 
+OMF - set to true if you have OMF enabled in your PIWEBAPI configuration tool, otherwise set this to false for legacy systems. See more info on OMF below.
```

Comment:
Set should be capitalized

Reply:
You can link to the section where you say "below" that way someone can just click the link and go directly there.